docs/guide/structure-applications.md

```diff
@@ -330,15 +330,15 @@ image size as a parameter like the following:
 ]
 ```
 
-Then in your code where you need to use the size value, you can simply use the code like the following:
+Then in your code where you need to use the size value, you can simply use code like the following:
```

Comment:
nothing special
purpose of this remark to save same style in document 
all other word-combinations "like the following" was replaced with "as follows" except this one
this is not critical so may be ignored